### ISSUE TITLE
Error HRESULT: 0x80073CF6

### ISSUE THREAD

**Issue by Leshu7w7** [LSPosed/MagiskOnWSA#139]
when running the script I get the following output: 
![image](https://user-images.githubusercontent.com/70908636/146625359-808c9882-17df-45bc-ab0d-7ca22581dcdf.png)

I decided to run every command from the script manually as Admin and I get an error when trying to add the appx xaml 
running `Add-AppxPackage -ForceApplicationShutdown -ForceUpdateFromAnyVersion -Path xaml.appx` output:
![image](https://user-images.githubusercontent.com/70908636/146625625-25628971-6a52-4bb2-a50b-94e455ea397f.png)

**xaml.appx** doesn't exist in the folder I guess that's the reason but I don't know how to get that file or solve the issue

**Comment by yujincheng08** [comment]
I can find `xaml.appx` in your zip from here: https://github.com/Leshu7w7/MagiskOnWSA/actions/runs/1594086688

**Comment by Leshu7w7** [comment]
But in the .zip artifact there's no file called xaml.appx
![image](https://user-images.githubusercontent.com/70908636/146644888-83e41182-fee2-4386-8c3e-939a10e322ed.png)

**Comment by yujincheng08** [comment]
I download the zip here and there is https://github.com/Leshu7w7/MagiskOnWSA/actions/runs/1594086688

**Comment by yujincheng08** [comment]
![image](https://user-images.githubusercontent.com/5022927/146645056-522bf50d-539f-4d88-a142-177134231b4d.png)
Apparently, your zip has missed many files. Please redownload.

**Comment by Leshu7w7** [comment]
ok, I'll redownload it and try again, thanks
if it works or not I'll tell you

**Comment by Ermiakel** [comment]
![image](https://user-images.githubusercontent.com/96384693/146734352-7c2ee6b7-405c-4cb2-afdb-a69429b1d0bd.png)
😭😭
i download this 3 time but same

**Comment by yujincheng08** [comment]
@Ermiakel  Apparently the size of `userdata.vhdx` should not be 0. Please check your connection.

**Comment by Leshu7w7** [comment]
thank you, it worked like a charm! 
first time I downloaded I was using a downloader manager (IDM) so that was the reason, I redownloaded without using a download manager and the missing files were there so once again thank you! 

P.S. At installing, when the script try to kill/stop wsaclient process it always gave me an error, the process didn't exist I just ignored it and continue the installation, but now I have to open some android apps two times, because the first time it crashed, it happened sometimes with play store, archero (a game) and tiktok lite, I haven't downloaded anything else yet so I can't say the same about another apps, did I do something wrong or is it normal due to wsa is in beta?
